pkg/tests/end_to_end_tests/database_metrics_test.go

```diff
@@ -50,7 +52,9 @@ func TestDatabaseMetrics(t *testing.T) {
 		compressionStatus = getMetricValue(t, "compression_status")
 		require.Equal(t, float64(1), compressionStatus)
 		numMaintenanceJobs = getMetricValue(t, "worker_maintenance_job")
-		require.Equal(t, float64(2), numMaintenanceJobs)
+		require.GreaterOrEqual(t, numMaintenanceJobs, float64(1))
+		metricsJobDuration = getMetricValue(t, "worker_maintenance_job_metrics_compression_last_duration_seconds")
```

Comment:
Can we check this metric is non zero after running `compress_chunk`. 
 Here https://github.com/timescale/promscale/pull/1745/files#diff-4517fe19b15a219db0b35419034d9c69c1fe5316a8d0a1b9db75579fed381ab6L137

Reply:
But it won't. It's only counting our own jobs and only when they run on schedule. AFAICT `compress_chunk` won't go through it.

Reply:
I see....If we could do some trick to make it schedule immediatelly? maybe wait a second and check the output it would be great. Anyway if it's too much effort I am fine with this.

Reply:
I've tried, and the obvious way, sadly, doesn't work, I'll leave it as is.